sogs/routes/users.py

```diff
@@ -76,16 +76,89 @@ def extract_rooms_or_global(req, admin=True):
     return (None, True)
 
 
-def _serialize_message(msg):
+def _serialize_message(msg, *, key):
     return {
         "id": msg.id,
         "posted_at": msg.posted_at,
         "expires_at": msg.expires_at,
         "message": utils.encode_base64(msg.data),
-        "sender": msg.sender.session_id,
+        key: getattr(msg, key).session_id,
     }
```

Comment:
I think we should just include both sender and recipient for both inbox/outbox messages.

Reply:
yea